### PR TITLE
Mark start node for translation

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link #5990 

Start node with various text lengths:

Screen Shot 2021-02-17 at 2 38 22 PM
Screen Shot 2021-02-17 at 2 37 53 PM
Screen Shot 2021-02-17 at 2 37 18 PM

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by AlexSCorey** [comment]
@mabashian  lets remove start from https://github.com/ansible/awx/blob/40cf4ff2090e4bc8816b05963bd76b665ed3ecd1/awx/ui_next/.eslintrc#L82

File: awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/VisualizerLink.jsx [hunk 1]
```diff
@@ -94,7 +94,8 @@ function VisualizerLink({
         ];
 
   const handleLinkMouseEnter = () => {
-    ref.current.parentNode.appendChild(ref.current);
+    const startNode = document.getElementById('node-1');
+    ref.current.parentNode.insertBefore(ref.current, startNode);
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/VisualizerLink.jsx:98]: I think remember translation being tricky for workflow nodes (?) in the old ui - glad to have found an approach here :+1:

**Review by jakemcdermott — APPROVED** [review]

**Review by jakemcdermott — APPROVED** [review]
I think remember translation being tricky for workflow nodes (?) in the old ui - glad to have found an approach here +1

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
See https://github.com/ansible/awx/pull/9344#issuecomment-780833299

**Comment by unlikelyzero** [comment]
@mabashian this will just need 🟢 from the e2e tests. We have coverage

**Review by AlexSCorey — APPROVED** [review]
Looks good to me

**Comment by unlikelyzero** [comment]
Unit test coverage is great! Marking this issue as needs_test and pending_translations so that we can test when real translations land

**Pull request merged**